zone/spell_effects.cpp

```diff
@@ -6275,9 +6275,9 @@ uint16 Client::GetSympatheticFocusEffect(focusType type, uint16 spell_id) {
 				if (SympatheticProcList.size() > MAX_SYMPATHETIC_PROCS)
 					continue;
 
-				EQ::ItemInstance *aug = nullptr;
+				EQ::ItemInstance const* aug = nullptr;
```

Comment:
Why is this a const here when value changes one line later? Seems like it’s unnecessary to not define the variable as the value the next line sets it to.

Reply:
Cleaned this up, thanks!